doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -753,7 +753,10 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
+  * Added the Kconfig option  :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_CHUNK_SZ` to process traces in chunks. This can improve the availability of trace memory, and thus reduce the chances of losing traces.
```

Comment:
```suggestion
  * Added the Kconfig option  :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_CHUNK_SZ` to process traces in chunks. 
    This can improve the availability of trace memory, and thus reduce the chances of losing traces.
```

Reply:
Added line break